pubspec.yaml

```diff
@@ -10,11 +10,11 @@ environment:
 dependencies:
   flutter:
     sdk: flutter
+  http: ^0.12.0+4
```

Comment:
[help] I don't understand why this is changed? Is it only for the http status constant?